internal/flag/flags.go

```diff
@@ -44,6 +44,7 @@ const (
 	AuthenticationEnabled                     = "authenticationEnabled"                     // AuthenticationEnabled flag
 	AuthorizationEnabled                      = "authorizationEnabled"                      // AuthorizationEnabled flag
 	Provider                                  = "provider"                                  // Provider flag
+	ProviderShort                             = "p"                                         // ProviderShort flag
```

Comment:
we already use `p` for password I would be hesitant to override it specially since every where else where we do provider we haven't added a shorthand

Reply:
Sure, I'll unpick this change then :+1:

Reply:
Resolved in [86a63c9](https://github.com/mongodb/mongodb-atlas-cli/pull/2138/commits/86a63c9b05cf5ea823d7221db29b630b2d081034)